app/views/wiki/show.html.erb

```diff
@@ -43,7 +43,7 @@
       <ul class="nav nav-tabs">
         <li class="active"><a href="#"><span class="hidden-xs"><%= t('wiki.show.view') %></span><span class="visible-xs"><i class="fa fa-file-o"></i></span></a></li>
         <% if @node.has_tag('locked') %>
-          <li><a href="/wiki/locked"><i class="fa fa-lock"></i></a></li>
+          <li><a href="/wiki/edit/pagetitle"><i class="fa fa-lock"></i></a></li>
```

Comment:
Oops, sorry -- `pagetitle` will actually have to be `<%= @node.edit_path %>?t=<%= Time.now.to_i %>` -- i was typing too fast and wasn't clear enough. Make sense?